pulsar-broker/src/test/java/org/apache/pulsar/client/impl/ServiceUrlQuarantineTest.java

```diff
@@ -53,10 +53,11 @@ public class ServiceUrlQuarantineTest extends ProducerConsumerBase {
     private PulsarClientImpl pulsarClientWithBinaryServiceUrlDisableQuarantine;
     private PulsarClientImpl pulsarClientWithHttpServiceUrl;
     private PulsarClientImpl pulsarClientWithHttpServiceUrlDisableQuarantine;
-    private static final int BROKER_SERVICE_PORT = 6666;
-    private static final int WEB_SERVICE_PORT = 8888;
+    private final int brokerServicePort = PortManager.nextLockedFreePort();
+    private final int webServicePort = PortManager.nextLockedFreePort();
```

Comment:
Set these in the setup method. We have retries for TestNG to address flakiness so that's why it's better to handle it that way.